src/odemis/acq/move.py

```diff
@@ -35,19 +34,68 @@
 from odemis.util import executeAsyncTask
 
 MAX_SUBMOVE_DURATION = 60  # s
-LOADING, IMAGING = 0, 1
+LOADING, IMAGING, TILTED, COATING, IMAGING_PATH, COATING_PATH, UNKNOWN = 0, 1, 2, 3, 4, 5, 6
```

Comment:
Since both movements to imaging and coating positions starts from the loading position, renamed `LOADING_PATH` to `IMAGING_PATH` to better reflect the destination.